### PR TITLE
Fixed torch.nn.MultiMarginLoss equation format error

### LINKED ISSUE #58634: torch.nn.MultiMarginLoss：equation format error
## 📚 Documentation
The error is in torch.nn.MultiMarginLoss
> For each mini-batch sample, the loss in terms of the 1D input x and scalar output y is:

`loss(x, y) = ∑ max(0,margin−x[y]+x[i]))^p / x.size(0)`

An extra parentheses is added to the right side of the equation
 
​	

cc @brianjo @mruberry @albanD @jbschlosser

### PR DESCRIPTION
**Pull request by aflah02**
Removed the extra parenthesis from the right side
Fixes #58634

### CODE REVIEW

**Comment by aflah02** [comment]
Hey @albanD @jbschlosser I'm not sure why one of the build fails. What's going wrong?

**Review by jbschlosser — APPROVED** [review]
lgtm! thanks

**Comment by aflah02** [comment]
> lgtm! thanks

You're welcome!
